.github/workflows/main.yml

```diff
@@ -5,11 +5,13 @@ on:
     branches:
       - main
     paths-ignore:
+      - .cruft.json
       - CHANGES.rst
       - README.rst
       - pyproject.toml
       - setup.cfg
       - setup.py
+      - tests/test_xscen.py
```

Comment:
Why ?

Reply:
There is one test that changes when we run `bump-my-version`. We shouldn't need to restart our tests when we bump the version, but maybe that's a good thing to do? I'm open to removing this.

Reply:
@aulemahal 

![image](https://github.com/Ouranosinc/xscen/assets/10819524/d0cdf7e6-c83f-4bf6-8860-1ba5d7b70f0b)

Reply:
Hmm I see!

I suggest removing either:
- the version number from the tested line
- the test file itself.

I don't see the point of that "smoke test" now that we have a minimal test suite. 
 @RondeauG  ?

Reply:
I agree that we could remove that test if it complicates things!

Reply:
Smoke tests removed!